### PR TITLE
Added side panels

### PR DESCRIPTION
**Pull request by Yeriwyn**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
V0 Magnetic panels

#### How was it tested? 
Tested on my machine

#### Any background context you want to provide?
Some users had requested magnetic side panels as well, it was a great idea!

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod update!

Can you modify `printer_mods/README.md` and change the title/description of your mod to reflect the fact it is not just the front panel?  If you wanna rename your directory from `V0_Magnetic_Front_Panel` that's fine too, but not necessary.

What's the difference between the front panel and side panels?

**Comment by Yeriwyn** [comment]
Thanks Kilo, I wanted to rename the directory but wasn't sure if that would have caused problems or not. Fixed now. 

The difference between the front and sides is the geometry of the top bar, the acrylic in the front extends further up than the ones on the sides do

File: printer_mods/README.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ like so:
 | | [V0 PiCam Mount](./xbst_/V0_Picam)| A Raspberry Pi Camera Mount For V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | | [DuetZero](./xbst_/DuetZero) | Pi Zero mount over a Duet2 for space saving | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:x:|
 | Xceled | [Single Panel Magnet Mount](./xceled/Single_Panel_Magnet_Mount)| Mount your front panel using only magnets | :x: |:grey_question: |:heavy_check_mark: |:x:|
-| Yeri | [V0 Magnetic Front Panel](./yeri/V0_Magnetic_Front_Panel)| Mount V0 front panel with magnets, removes need for hinges | :heavy_check_mark: | :x: | :x: | :x: |
+| Yeri | [V0 Magnetic Panels](./yeri/V0_Magnetic_Panels)| Mount V0 front panel with magnets, removes need for hinges | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:213]: Update the description too, it still says `Mount V0 front panel with magnets`...

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks!  Minor README fix, and it looks like your branch now has merge conflicts, can you merge upstream into your branch and fix the conflicts?

```
Conflicting files
printer_mods/yeri/V0_Magnetic_Front_Panel/Mounting_Plate_v2.stl
printer_mods/yeri/V0_Magnetic_Front_Panel/Panel_Mount_v2.stl
```

**Comment by Yeriwyn** [comment]
checking the merge conflicts...

**Comment by Yeriwyn** [comment]
Ok I think that's got it cleared now

**Review by KiloQubit — APPROVED** [review]
Awesome, thanks!

**Pull request merged**